### ISSUE TITLE
Uploading files

### ISSUE THREAD

**Issue by ghost** [Neo11/ChitChat#10]
Is it possible to add a feature for uploading files? So I can set a group icon?

Best Regards,

Jules

**Comment by c-o-m-m-a-n-d-e-r** [comment]
I will look at it, but i think it's not so easy to do that ...

**Comment by ghost** [comment]
Thank you very much!

**Comment by tomatoTom** [comment]
+1 this would be nice @c-o-m-m-a-n-d-e-r 
a download function would be also nice :+1:

**Comment by c-o-m-m-a-n-d-e-r** [comment]
a download function? what do you mean exactly?

**Comment by Neo11** [comment]
I think with a _WKScriptMessageHandler_ called instead of _alert_ the upload should be possible

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes but i don't know if the really support uploading Group Chat Avatars through Web Version ...

**Comment by tomatoTom** [comment]
In ChitChat is a download button when you are opening a media file. When you click on the button the file should be saved to the desktop. This would be also a nice feature.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
Ah okay yes now i see ... :-) ... will take a look too ..
I think this could be possible

**Comment by tomatoTom** [comment]
Thank you @c-o-m-m-a-n-d-e-r  :+1:

**Comment by ghost** [comment]
Hello everyone, are there any news? Do you think it's possible?

Have a nice day!

**Comment by c-o-m-m-a-n-d-e-r** [comment]
not an easy task... can't give a timeline for that sry!

**Comment by VagishVela** [comment]
Proper media upload is working now: 
https://bugs.webkit.org/show_bug.cgi?id=137759
